### ISSUE TITLE
backbone weight names?

### ISSUE THREAD

**Issue by BuaaAlban** [bubbliiiing/ssd-tf2#12]
I have a question about how do you get the pretrained backbone vgg (You pretrained it by yourself with your model or get it from other sources)? 
And how can I align the weight names from other public pretrained model to the current one

**Comment by bubbliiiing** [comment]
imagnet？

**Comment by BuaaAlban** [comment]
Yes, pretrained model on imagenet. I want to replace vgg by resnet34 as backbone.
Do you pretrain the vgg on imagenet by yourself?

**Comment by bubbliiiing** [comment]
There is pretrained model in TORCHVISION

**Comment by BuaaAlban** [comment]
Yes, but it's in pytorch version, though I can do the mapping of names

**Comment by bubbliiiing** [comment]
it's same, i get it from torchvision
